### DISCUSSION TITLE
I'm receiving this message "The node is temporarily unusable due to resource exhaustion" What does it mean?

### DISCUSSION THREAD

**Discussion by N0tNic0** [globaleaks/globaleaks-whistleblowing-software#4551] [Q&A] [answered]
After starting the Globaleaks process from the terminal on linux, the following error appears:

![globaleaks_error_00](https://user-images.githubusercontent.com/95221916/164015107-d3410d65-2203-4219-93d2-58c93072470c.PNG)

while accessing the web application the following error is displayed:

![globaleaks_error_01](https://user-images.githubusercontent.com/95221916/164015670-9c1aa75a-3e91-4735-a293-656e46f87a04.PNG)

**Desktop (please complete the following information):**
 - OS: Ubuntu 16.04.4 LTS
 - Browser: ALL
 - Version 3.1.6

**Comment by evilaliv3** [comment, marked as answer]
Dear @N0tNic0: The reason why you are receiving this message is that your disk is probably full.

Please check the disk allocation and instantiate more disk space.

Please check as well to have a reasonable data retention policy to delete old reports automatically.

I notice that you are running on a very oudated version of globaleaks; To ensure the system is secure i invite you to  update both the operating system to ubuntu 20.04 and GlobaLeaks to 4.9.2.
